File: Assignment_No12/File3.c
#include<stdio.h>

int CountRange(int iNo)
{
    int iCount=0, iDigit=0, iFrequency=0;

    while(iNo != 0)
    {
        iDigit= iNo%10;

        if(iDigit>3 && iDigit<7)
        {
            iFrequency++;
        }
        iNo = iNo/10;
    }
    return iFrequency;
}

int main()
{
    int iValue=0, iRet=0;

    printf("Enter a number \n");
    scanf("%d",&iValue);

    iRet = CountRange(iValue);

    printf("%d",iRet);

    return 0;
}

File: Assignment_No34/File3.java
import java.util.*;
import java.io.*;

class File3
{
    public static void main(String arg[])
    {
        int iNo = 0;

        try
        {
            BufferedReader Input = new BufferedReader(new InputStreamReader(System.in));

            System.out.println("Please enter element");
            iNo = Integer.parseInt(Input.readLine());

            Digit dObj = new Digit();

            System.out.println("The count of even digits: " + dObj.CountRange(iNo));
        }
        catch(Exception eobj)
        {
            System.out.println(eobj);
        }
    }
}

class Digit
{
    public int CountRange(int iValue)
    {
        int iDigit = 0, iCountRange = 0;
        while(iValue != 0)
        {
          iDigit = iValue % 10; 
        
            if(iDigit > 3 && iDigit < 7)
            {
                iCountRange++;
            }
          iValue = iValue/10;
        }
        return iCountRange;
    }
}